clients/vscode-hlasmplugin/src/hlasmConfigurationDiagnosticsProvider.ts

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright (c) 2023 Broadcom.
+ * The term "Broadcom" refers to Broadcom Inc. and/or its subsidiaries.
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *   Broadcom, Inc. - initial API and implementation
+ */
+
+import * as vscodelc from 'vscode-languageclient';
+
+export function toggleNonCriticalConfigurationDiagnostics(client: vscodelc.BaseLanguageClient) {
+    client.sendRequest<void>("toggle_non_critical_configuration_diagnostics");
```

Comment:
I think this should be sendNotification - we are not expecting answer and the server does not provide one.